extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php

```diff
@@ -79,7 +78,6 @@ private function getWikisWithWatchedPagesForUser( $userID ) {
 			->SELECT()->DISTINCT( GlobalWatchlistTable::COLUMN_CITY_ID )
 			->FROM( GlobalWatchlistTable::TABLE_NAME )
 			->WHERE( GlobalWatchlistTable::COLUMN_USER_ID )->EQUAL_TO( $userID )
-			->AND_( GlobalWatchlistTable::COLUMN_TIMESTAMP )->IS_NOT_NULL()
```

Comment:
This query was incorrect, we don't insert rows in the `global_watchlist` table with a null timestamp.